### DISCUSSION TITLE
Introduction and Enquiry

### DISCUSSION THREAD

**Discussion by Trailblazer007** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#44] [Outreachy 2022] [answered]
Hello everyone and the mentors too @BatoolMM and @alswajiab. I'm Seun from Nigeria. I'd like to be a part of this project too.
Thank you.

**Comment by BatoolMM** [comment, marked as answer]
Welcome, @Trailblazer007  to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
